docs/configuration-reference/backend/s3.md

```diff
@@ -19,8 +19,23 @@ used.
 
 ## Prerequisites
 
-* AWS S3 bucket to be used should already be created.
+* AWS S3 bucket to be used should already be created. This can be done through the following aws CLI command:
+
+```
+aws s3 mb s3://kinvolk-lokomotive-test --region=eu-central-1
+```
+
 * DynamoDB table to be used for state locking should already be created. The table must have a primary key named LockID.
+This can be established with the `aws dynamodb create-table` command:
```

Comment:
I find the word "established" weird here. Also you can apply the more direct style and be more consistent with the previous point. How about:

```suggestion
You can create it with the following AWS CLI command:
```